src/services/widgets/widgets.svc.coffee

```diff
@@ -87,11 +83,11 @@ angular
             # The widget's metadata are updated only if the correct setting is attached to the widget
             if settingKey in wgtSettingsKeys
               angular.extend wgt.metadata, setting.toMetadata()
-              wgt.isLoading = true
-              _self.update(wgt, { metadata: wgt.metadata }).then(
-                (updatedWidget) ->
-                  _self.show(updatedWidget).finally( -> updatedWidget.isLoading = false )
-              )
+              promises.push _self.update(wgt, { metadata: wgt.metadata })
```

Comment:
Is the `.then((updatedWidget)-> updatedWidget.isLoading = false ` not needed anymore?